src/OfflineSearch.xml

```diff
@@ -0,0 +1,51 @@
+<?xml version="1.0" encoding="utf-8"?>
+<widget id="com.mendix.widget.custom.OfflineSearch.OfflineSearch" offlineCapable="true" needsEntityContext="false" xmlns="http://www.mendix.com/widget/1.0/">
+    <name>Offline Search</name>
+    <description>Search Mendix List views, Template Grids and Data Grids using a single-field search</description>
+    <icon />
+
+    <properties>
+        <property key="targetGridName" type="string" required="true" defaultValue="">
+            <caption>Grid name</caption>
+            <category>Data source</category>
+            <description>The name of the grid on this page</description>
+        </property>
+        <property key="searchMethod" type="enumeration" defaultValue="contains">
+            <caption>Search method</caption>
+            <category>Data source</category>
+            <description>Search method to use</description>
+            <enumerationValues>
+                <enumerationValue key="equals">Equals</enumerationValue>
+                <enumerationValue key="lessThan">Less than</enumerationValue>
+                <enumerationValue key="lessThanOrEquals">Less than or equals</enumerationValue>
+                <enumerationValue key="greaterThan">Greater than</enumerationValue>
+                <enumerationValue key="greaterThanOrEquals">Greater than or equals</enumerationValue>
+                <enumerationValue key="contains">Contains</enumerationValue>
+            </enumerationValues>
+        </property>
+        <property key="searchEntity" type="string" required="false">
+            <caption>Grid entity</caption>
+            <category>Data source</category>
+             <description>The type of objects in the grid</description>
+        </property>
+        <property key="searchAttribute" type="string" required="false">
+            <caption>Search attribute</caption>
+            <category>Data source</category>
+            <description>The search attribute on the search entity</description>
+        </property>
+        <property key="defaultQuery" type="translatableString" required="false">
+            <caption>Default query</caption>
+            <category>Data source</category>
+            <description>Limits the records show on the grid at the start</description>
+        </property>
+        <property key="showSearchBar" type="enumeration" defaultValue="visible">
+            <caption>Show search bar</caption>
+            <category>Appearance</category>
+            <description> Use to control if the search bar should be visible or not</description>
```

Comment:
Remove space `> U`
Explain when the search bar is required?
Would this be a Boolean?